buildozer/default.spec

```diff
@@ -114,19 +114,23 @@ fullscreen = 0
 # (bool) If True, then skip trying to update the Android sdk
 # This can be useful to avoid excess Internet downloads or save time
 # when an update is due and you just want to test/build your package
-# android.skip_update = False
+#android.skip_update = False
```

Comment:
don't touch theses lines since it's out of the context of the PR. And actually that's the reason the CI is failing:
```
The following packages can not be installed since their licenses or those of the packages they depend on were not accepted:
```

Reply:
ok. put it back.. very strange